governance/chair-proposal-process.md

```diff
@@ -0,0 +1,7 @@
+# SIG-Security Chair Proposal Process
+
+1) SIG-Security Co-chairs operate as a team. The Co-Chair team seeks to maintain full coverage of the leadership capabilities across the key qualities of industry experience, hands-on cloud-native and security experience, as well as administrative experience needed to run a SIG-Security.
```

Comment:
nit: noticed inconsistency in caps for "Co-chairs" and checked https://github.com/cncf/sig-security/blob/master/governance/roles.md#role-of-chairs -- seems we just use "Chair" in the role name.  I find that more readable